testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketConnectionAttemptTest.java

```diff
@@ -96,7 +95,7 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
 
         cb.handler(handler);
         cb.option(ChannelOption.ALLOW_HALF_CLOSURE, halfClosure);
-        ChannelFuture future = cb.connect(NetUtil.LOCALHOST, TestUtils.getFreePort()).awaitUninterruptibly();
+        ChannelFuture future = cb.connect(NetUtil.LOCALHOST, BAD_PORT).awaitUninterruptibly();
```

Comment:
The problem with crawling is that it may be possible the server bind the port in between. Let me use 4 for now.